app/views/devise/passwords/new.html.slim

```diff
@@ -4,12 +4,14 @@
   .medium-6.medium-centered.large-4.large-centered.columns
     .row.column.log-in-form
       h2.text-center
-        | Forgot your password?
+        | Forgot Your Password?
       = simple_form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :post }) do |f|
         = f.error_notification
         .form-inputs
           = f.input :email, required: true, autofocus: true
+        br
```

Comment:
yup so that way if you want to change the the spacing later you can and it'll effect the hole site.